contracts/script/Deployer.s.sol

```diff
@@ -0,0 +1,16 @@
+// SPDX-License-Identifier: UNLICENSED
```

Comment:
This is a lightly modified version of the default script that comes with Foundry. You can ignore for now, it's not being used yet.

Would like to eventually use it so that we can get consistent contract addresses via [Create2](https://docs.openzeppelin.com/cli/2.8/deploying-with-create2) instead of having them change on every deploy.